cvxpylayers/torch/cvxpylayer.py

```diff
@@ -298,6 +289,10 @@ def backward(ctx, *dvars):
 
             if not ctx.batch:
                 grad = [g.squeeze(0) for g in grad]
+            else:
+                for i, (g, sz) in enumerate(zip(grad, ctx.batch_sizes)):
+                    if sz == 0:
+                        grad[i] = grad[i].sum(dim=0)
```

Comment:
grad[i].squeeze(0)?

Reply:
It should be a sum across the batch dimension here since the shared/non-batched parameter needs to accumulate the gradients from all of the batch instances

Reply:
gotcha